Makefile

```diff
@@ -204,6 +204,21 @@ else ifeq ($(platform), switch)
 	STATIC_LINKING=1
 	fpic := -nostdlib
 
+# Nintendo Switch (libnx)
+else ifeq ($(platform), libnx)
+    include $(DEVKITPRO)/libnx/switch_rules
+    EXT=a
+    TARGET := $(TARGET_NAME)_libretro_$(platform).$(EXT)
+    DEFINES := -DSWITCH=1 -U__linux__ -U__linux -DRARCH_INTERNAL
+    CFLAGS  :=  $(DEFINES) -g \
+                -O2 \
```

Comment:
Please use O3 with -ftree-vectorize
Also remove the "\" from the lone O2
This statement will need to be changed too https://github.com/libretro/virtualjaguar-libretro/blob/bb43634b549e7a27ab92a6eebeaa9cb842222e64/Makefile#L526

Reply:
With statement I mean the if around that